src/main/java/mil/dds/anet/search/AbstractTaskSearcher.java

```diff
@@ -73,12 +73,17 @@ protected void addBatchClause(TaskSearchQuery query) {
     qb.addBatchClause((AbstractBatchParams<Task, TaskSearchQuery>) query.getBatchParams());
   }
 
-  protected void addResponsibleOrgUuidQuery(TaskSearchQuery query) {
+  protected void addTaskedOrgUuidQuery(TaskSearchQuery query) {
+
+    qb.addFromClause(
+        "LEFT JOIN taskTaskedOrganizations ON task.uuid = taskTaskedOrganizations.\"taskUuid\"" +
+        " JOIN organizations ON taskTaskedOrganizations.\"organizationUuid\" = organuzations.uuid");
```

Comment:
Properly quote camel-cased tables and columns; also a typo in `organuzations`.

Reply:
You probably don't need the join on organizations, if you use `\"taskTaskedOrganizations\".\"organizationUuid\"` in the query stanzas below.